### PR TITLE
🧹 BL3 Testnet Config

### PR DESCRIPTION
**Pull request by ravinagill15**
## In this PR:
- Updated dependencies to retrieve BL3 eid
- Updated configs to add BL3

### CODE REVIEW

File: packages/stg-definitions-v2/src/constant.ts [hunk 1]
```diff
@@ -360,6 +365,10 @@ export const ASSETS: Record<TokenName, AssetConfig> = {
             [EndpointId.ARBSEP_V2_TESTNET]: {
                 type: StargateType.Pool,
             },
+            [EndpointId.BL3_V2_TESTNET]: {
+                type: StargateType.Oft,
+                address: 'RAVINA TODO',
```
Comment (ravinagill15) [inline packages/stg-definitions-v2/src/constant.ts:370]: Will update soon!

File: packages/stg-definitions-v2/src/constant.ts [hunk 2]
```diff
@@ -503,6 +512,10 @@ export const ASSETS: Record<TokenName, AssetConfig> = {
                 address: '0x3253a335E7bFfB4790Aa4C25C4250d206E9b9773',
                 type: StargateType.Pool,
             },
+            [EndpointId.BL3_V2_TESTNET]: {
+                type: StargateType.Oft,
+                address: 'RAVINA TODO',
```
Comment (ravinagill15) [inline packages/stg-definitions-v2/src/constant.ts:517]: Will update soon!

**Comment by ravinagill15** [comment]
testnet build failing is expected -- now that we are deployed an _external_ USDC proxy, it's master minter has already been configured correctly once and cannot really be tested in ci/cd since it cannot be re-deployed over and over. I will think about the best way around this for now

File: packages/stg-evm-v2/devtools/config/testnet/feelib-v1.eth.config.ts [hunk 3]
```diff
@@ -44,83 +40,11 @@ export default async (): Promise<OmniGraphHardhat<FeeLibV1NodeConfig, FeeLibV1Ed
...
-            },
-            {
-                from: klaytnFeeLibV1,
-                to: optFeeLibV1,
-                config: defaultEdgeConfig,
-            },
-            {
-                from: klaytnFeeLibV1,
-                to: arbFeeLibV1,
-                config: defaultEdgeConfig,
```
[the remark this answers is not in the document]
Comment (calebbanister) [reply packages/stg-evm-v2/devtools/config/testnet/feelib-v1.eth.config.ts:122]: ok

**Review by calebbanister — APPROVED** [review]

**Pull request merged**